packages/webpack/src/plugins/vue/cors.js

```diff
@@ -7,15 +7,16 @@ export default class CorsPlugin {
     const ID = `vue-cors-plugin`
     compiler.hooks.compilation.tap(ID, (compilation) => {
       compilation.hooks.htmlWebpackPluginAlterAssetTags.tap(ID, (data) => {
-        if (this.crossorigin != null) {
-          [...data.head, ...data.body].forEach((tag) => {
-            if (tag.tagName === 'script' || tag.tagName === 'link') {
-              if (tag.attributes) {
-                tag.attributes.crossorigin = this.crossorigin
-              }
-            }
-          })
+        if (!this.crossorigin) {
```

Comment:
As crossorigin can only have `anonymous` or `use-credentials` as values ([source](https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_settings_attributes)), I directly checked for a falsy value

Reply:
LGTM :+1: